Modules/Quest/QuestieQuest.lua

```diff
@@ -605,6 +605,13 @@ function QuestieQuest:AddFinisher(quest)
                             Name = finisher.name,
                             IsObjectiveNote = false,
                         }
+                        if QuestieDB.IsActiveEventQuest(quest.Id) then
+                            data.Icon = Questie.ICON_TYPE_EVENTQUEST_COMPLETE
+                        elseif QuestieDB.IsPvPQuest(quest.Id) then
```

Comment:
Random though: Might be worth to add a caching to `QuestieDB.IsPvPQuest`, `QuestieDB.IsActiveEventQuest`, etc. since it potentially is run quite often 🤷🏻 (not now, but we could create an issue, if you agree that this is a good idea)